labgrid/util/ssh.py

```diff
@@ -525,6 +525,10 @@ def disconnect(self):
                 self._logger.info("Closing SSH connection to %s", self.host)
                 self._stop_keepalive()
                 self._stop_own_master()
+
+            # stop keepalive when existing connection is used
+            if self._keepalive:
+                self._stop_keepalive()
```

Comment:
Why not stop the keepalive first? That would avoid the special case and duplicated `self._stop_keepalive()`?